### PR TITLE
Change Phabricator example to use v1

### PR DESCRIPTION
**Pull request by krousey**
#9286

@caesarxuchao @bgrant0607 @nikhiljindal

### CODE REVIEW

File: examples/phabricator/README.md [hunk 1]
```diff
@@ -188,10 +188,7 @@ Use the file [`examples/phabricator/phabricator-service.json`](phabricator-servi
     "selector": {
       "name": "phabricator"
     },
-    "createExternalLoadBalancer": true,
-    "publicIPs": [
-      "107.178.210.6"
-    ]
+    "type": "LoadBalancer"
```
Comment (caesarxuchao) [inline examples/phabricator/README.md]: I think we should add instructions on how to get the Load Balancer's IP address, like [this](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/guestbook-go#step-six-create-the-guestbook-service)

**Comment by caesarxuchao** [comment]
A nit, otherwise LGTM. Thanks.

**Comment by krousey** [comment]
Done.

**Comment by caesarxuchao** [comment]
LGTM

**Pull request merged**